arangod/Aql/grammar.h

```diff
@@ -1,4 +1,4 @@
-/* A Bison parser, made by GNU Bison 3.4.1.  */
```

Comment:
why was this committed ?

Reply:
By mistake. It is a generated file anyway, so it doesn't do any harm.